tests/bench/refusalbench_common.py

```
@@ -0,0 +1,541 @@
+    print(f"Total Items: {metrics['total']}")
+    print(f"Correct: {metrics['correct']}")
+    print(f"Overall Accuracy: {metrics['accuracy']:.3f}")
+    print(f"Calibrated Refusal Score (CRS): {metrics['calibrated_refusal_score']:.3f}")
+    print(f"Total Time: {format_duration(total_duration)}")
+
+    print(f"\n{'─' * 40}")
+    print("Answerable Items:")
+    print(f"  Total: {metrics['answerable_total']}")
+    print(f"  False Refusal Rate: {metrics['false_refusal_rate']:.3f}")
+
+    print(f"\nUnanswerable Items:")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)